app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,34 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.includes(:comments)
-    # puts @posts
   end
 
   def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+    @user = User.find(params[:user_id])
+    @current_user = current_user
+  end
+
+  def post_params
+    params.require(:post).permit(:title, :text)
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.build(post_params)
+    @post.comments_counter = 0
+    @post.likes_counter = 0
+    @post.author_id = current_user.id
```

Comment:
[OPTIONAL] Doing something like this would have been more straightforward:
`@post = @current_user.posts.new(post_params)`